src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { PaginationLeftItem, PaginationRightItem, PaginationPanel } from './styled'
+import LeftBlack from '../../assets/pagination_black_left.png'
+import RightBlack from '../../assets/pagination_black_right.png'
+import i18n from '../../utils/i18n'
+
+const PaginationItem = ({
+  currentPage,
+  total,
+  pageSize,
+  defaultJumpPage,
+  onChange,
+}: {
+  currentPage: number
+  total: number
+  pageSize: number
+  defaultJumpPage: number
+  onChange: (page: number, pageSize: number) => void
+}) => {
+  const [inputValue, setInputValue] = useState(defaultJumpPage)
+  const totalPage = Math.ceil(total / pageSize)
+  const goFirstPage = () => {
+    onChange(1, pageSize)
+  }
+
+  const goLastPage = () => {
+    onChange(totalPage, pageSize)
+  }
+
+  const goPrev = () => {
+    if (currentPage > 1) {
+      onChange(currentPage - 1, pageSize)
+    }
+  }
+
+  const goNext = () => {
+    if (currentPage < totalPage) {
+      onChange(currentPage + 1, pageSize)
+    }
+  }
+
+  const gotoPage = () => {
+    if (inputValue && inputValue <= totalPage && inputValue >= 1) {
+      onChange(inputValue, pageSize)
+    }
+  }
+
+  const isFirstPage = () => {
+    return currentPage === 1
+  }
+
+  const isLastPage = () => {
+    return currentPage === totalPage
+  }
+
+  return (
+    <PaginationPanel>
+      <PaginationLeftItem isFirstPage={isFirstPage()} isLastPage={isLastPage()}>
+        <button type="button" className="first" onClick={() => goFirstPage()}>
+          {i18n.t('pagination.first')}
+        </button>
+        <button type="button" className="left_image" onClick={() => goPrev()}>
```

Comment:
http://getbem.com/naming/, css name rule , lets keep same with this in this project